logstash-core/lib/logstash/config/mixin.rb

```diff
@@ -513,6 +514,12 @@ def validate_value(value, validator)
             end
 
             result = value.first.is_a?(::LogStash::Util::Password) ? value.first : ::LogStash::Util::Password.new(value.first)
+          when :uri
+            if value.size > 1
+              return false, "Expected uri (one value), got #{value.size} values?"
+            end
+            
+            result = value.first.is_a?(::LogStash::Util::SafeURI) ? value.first : ::LogStash::Util::SafeURI.new(value.first)
```

Comment:
> value.first.is_a?(::LogStash::Util::SafeURI)

When would this be true if `validate_value` is called on the configuration?

Currently, the only time I see this would be true if someone was to set a value as the default value, right?